source/_posts/2015-09-10-mqtt-different-ways-to-use-mqtt-with-home-assistant.markdown

```diff
@@ -0,0 +1,201 @@
+---
+layout: post
+title: "MQTT: Different ways to use MQTT with Home Assistant"
```

Comment:
Let's remove the `MQTT:` at the start. I always aim to keep the titles on 1 line on the [blog page](https://home-assistant.io/blog/)